view/frontend/requirejs-config.js

```diff
@@ -2,7 +2,17 @@ var config = {
     map: {
         '*': {
             KlaviyoCustomerData: 'Klaviyo_Reclaim/js/customer',
-            'Magento_Checkout/js/model/shipping-save-processor/default': 'Klaviyo_Reclaim/js/model/shipping-save-processor/default'
+        }
+    },
+    config: {
+        mixins: {
+            // Compatibility with Magento < 2.2.2
+            'mage/storage': {
```

Comment:
this is a nice solution for below 2.2.2 - i don't think it'll be required because the current version 3.0.5 + will only support 2.3.4+

Reply:
Thanks for the insights. I removed the 2.2.2 compatibility